### PR TITLE
Fix 404 links and update block codes format - orderForm

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [X] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

### CODE REVIEW

File: docs/guides/Checkout/orderform-fields.md [hunk 1]
```diff
@@ -171,29 +211,89 @@ Object containing the gift registration list data.
...
+                ],
+                “total”: 5100
+            }
+        }
+    ]
+```
+
 | Field      | Type      | Description |
 | ---------- | ---------- | ---------- |
 |    uniqueId |    String  |  This string is the unique identifier for each occurrence of a SKU in an order. Two same SKUs in an order will have different uniqueIds. Obsolete field.     |
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:299]:
```suggestion
|    uniqueId |    String  |  Obsolete field. This string is the unique identifier for each occurrence of a SKU in an order. Two units of the same SKU in an order will have different uniqueIds.    |
```

File: docs/guides/Checkout/orderform-fields.md [hunk 2]
```diff
@@ -303,16 +399,24 @@ it is an array containing an object describing every item in the buyer's cart.
...
+
+```json
 {
-  "codes": [
-    {
-      "code": "{\n   \"address\": {\n   \"postalCode\": \"******000\",\n   \"city\": \"Rio ** *******\",\n   \"state\": \"RJ\",\n   \"country\": \"BRA\",\n   \"street\": \"Rua *** *****nte\",\n   \"number\": \"***\",\n   \"neighborhood\": \"Bot*****\",\n   \"complement\": \"*** ** *\",\n   \"reference\": null\n  \t},\n    \"settleInvoices\":[]  \n}     ",
-      "language": "json"
-    }
-  ]
+   "address": {
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:405]: Corrigir indentação:

```json
{
  "address": {
    "postalCode": "******000",
    "city": "Rio ** *******",
    "state": "RJ",
    "country": "BRA",
    "street": "Rua *** *****nte",
    "number": "***",
    "neighborhood": "Bot*****",
    "complement": "*** ** *",
    "reference": null
  },
  "settleInvoices": []
}
```

File: docs/guides/Checkout/orderform-fields.md [hunk 3]
```diff
@@ -459,16 +642,23 @@ Information on rates and benefits that apply to the order.
 ### selectableGifts
 
 Array containing the data of the item selected as a gift.
-[block:code]
+
+```json
 {
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:647]:
```suggestion

```

File: docs/guides/Checkout/orderform-fields.md [hunk 4]
```diff
@@ -459,16 +642,23 @@ Information on rates and benefits that apply to the order.
...
+    "id": null,
+    "availableQuantity": "3",
+    "availableGifts": {
+       "items": [
+                 {},
+                 "isSelected": true
+                ]
+   }
+ ]
 }
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:659]:
```suggestion

```

File: docs/guides/Checkout/orderform-fields.md [hunk 5]
```diff
@@ -459,16 +642,23 @@ Information on rates and benefits that apply to the order.
...
+  {
+    "id": null,
+    "availableQuantity": "3",
+    "availableGifts": {
+       "items": [
+                 {},
+                 "isSelected": true
+                ]
+   }
+ ]
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:658]:
```suggestion
  }
 ]
```

File: docs/guides/Checkout/orderform-fields.md [hunk 6]
```diff
@@ -567,49 +885,61 @@ This object contains shipping information for the order.
...
-  ]
-}
-[/block]
\ No newline at end of file
+__Example:__
+
+```json
+[
+  {
+    "id": "Items"
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:936]:
```suggestion
    "id": "Items",
```

File: docs/guides/Checkout/orderform-fields.md [hunk 7]
```diff
@@ -567,49 +885,61 @@ This object contains shipping information for the order.
...
+    "name": "Total items"
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:937]:
```suggestion
    "name": "Total items",
```

File: docs/guides/Checkout/orderform-fields.md [hunk 8]
```diff
@@ -567,49 +885,61 @@ This object contains shipping information for the order.
...
+    "value": 35620
+  }, {
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:939]:
```suggestion
  },
  {
```

File: docs/guides/Checkout/orderform-fields.md [hunk 9]
```diff
@@ -567,49 +885,61 @@ This object contains shipping information for the order.
...
+    "id": "Shipping"
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:940]:
```suggestion
    "id": "Shipping",
```

File: docs/guides/Checkout/orderform-fields.md [hunk 10]
```diff
@@ -567,49 +885,61 @@ This object contains shipping information for the order.
...
+    "name": "Total freight"
```
Comment (julia-rabello) [inline docs/guides/Checkout/orderform-fields.md:941]:
```suggestion
    "name": "Total freight",
```

**Review by karenkrieger — COMMENTED** [review]
Mesmo comentários da @julia-rabello

**Review by karenkrieger — APPROVED** [review]
LGTM

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**